src/shared/ask-password-api.c

```diff
@@ -803,10 +803,10 @@ int ask_password_agent(
                 "Silent=%i\n",
                 getpid_cached(),
                 socket_name,
-                (flags & ASK_PASSWORD_ACCEPT_CACHED) ? 1 : 0,
-                (flags & ASK_PASSWORD_ECHO) ? 1 : 0,
+                FLAGS_SET(flags, ASK_PASSWORD_ACCEPT_CACHED) ? 1 : 0,
```

Comment:
you can drop the `? 1 : 0` thing, after all FLAGS_SET() always returns 1 or 0 anyway.

Reply:
Right you are... :)